src/pages/workspaces/workspace/WorkspaceContainer.js

```diff
@@ -257,7 +250,7 @@ const useCloudEnvironmentPolling = (googleProject, workspace) => {
         Ajax(signal).Runtimes.listV2(cloudEnvFilters)
```

Comment:
@jdcanas the cloudEnvFilters provided at [line 244](https://github.com/DataBiosphere/terra-ui/pull/3740/files#diff-1f45f11cb74b4b6ee8c1c20f21df6fedeebdf53163ac7589148d5f43867534d6R244) are confusing to me. For one thing, there is a check of `saturnWorkspaceVersion === 'v1'`, which I didn't think was information that was available. But beyond that, when I print the disks and runtimes being returned with those filters, I only get 1 thing being returned (for an Azure workspace) even though I actually have a runtime and persistent disk for a v2 GCP workspace as well.

Reply:
Oh… I think I understand. The disks and runtimes here are supposed to only be the ones linked to this particular workspace.

Reply:
Chatted with @jdcanas about this and deleted the `saturnWorkspaceVersion` code in https://github.com/DataBiosphere/terra-ui/pull/3740/commits/887988b502751e908ee72d14c15d64aae9a7390f.

I was assuming it was some Leo thing because of the "saturn" prefix, which is on the labels that come back from Leo. But it was just getting the workspace version from the workspace itself, and we won't be returning v1 workspaces anymore from Rawls so the version will never be v1.